homeassistant/components/foscam/__init__.py

```diff
@@ -1 +1,51 @@
 """The foscam component."""
+import asyncio
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_NAME
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
I'm not sure what do you mean by this, I tried searching around the development docs.

I've maintained that because the scaffolding of the config_flow integration added it, but I'm not really using it anywhere nor really know what is for. It was not present prior to my commit so if it needs to be deprecated I understand I could remove it directly, right?

Reply:
Ok then you don't really need it for your case. Then it's the other schema that you should deprecate

Reply:
Ok checking other integrations I got it. Since this one has no data to migrate I didn't add code to handle it, just marked it as deprecated and removed the code that stores data there and now I'm just using the entity registry.

Reply:
Ah ok, I have to deprectate too the platform schema will add it then.